### PR TITLE
fix: update horizontal pading for BalanceTokenCell

### PR DESCRIPTION
**Pull request by cythb**

### CODE REVIEW

File: Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift [hunk 1]
```diff
@@ -75,8 +75,8 @@ class BalanceTokenCell: UITableViewCell {
         containerView.addSubview(tokenBalanceLabel)
         
         NSLayoutConstraint.activate([
-            containerView.leadingAnchor.constraint(equalTo: contentView.readableContentGuide.leadingAnchor),
-            containerView.trailingAnchor.constraint(equalTo: contentView.readableContentGuide.trailingAnchor),
+            containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: LayoutConstraints.leading),
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -LayoutConstraints.trailing),
```
Comment (jk234ert) [inline Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift:79]: Why changes to fixed padding value? This would break the appearance on iPad

Comment (cythb) [reply Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift:79]: The UI designer told me to adjust the horizontal margin to 20pt, where it is not aligned with the rest. 
Or do I adjust all the places where the margins are 20 to `readableContentGuide`?

**Review by sxiaojian88 — APPROVED** [review]

File: Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift (hunk 1 above)
Comment (jk234ert) [reply Maskbook/Scene/Wallet/Balance/View/BalanceTokenCell.swift:79]: Plz adjust the padding with `readableContentGuide`

**Comment by cythb** [comment]
The `readableContentGuide` spacing size varies from device to device. For example, iPhone 8 is 16pt, while iPhone 13 pro is 20pt, so there is no good way to make the horizontal spacing 20pt here. later spacing is more appropriate according to `readableContentGuide`.

**Pull request merged**